pkgs/by-name/dp/dprint/plugins/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchurl }:
+let
+  mkDprintPlugin =
+    {
+      url,
+      hash,
+      pname,
+      version,
+      description,
+      initConfig,
+      updateUrl,
+    }:
+    fetchurl {
+      inherit hash url;
+      name = "${pname}-${version}.wasm";
+      meta.description = description;
```

Comment:
Will these packages have no maintainers? I suggest to add yourself :pray:

Reply:
done